iree/hal/cts/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@
 set(IREE_ALL_CTS_TESTS
   "allocator"
   "buffer_mapping"
+  "command_buffer_dispatch"
```

Comment:
is this order from a tool? I feel like root before root_suffix is normal?

Reply:
I thought I saw that ordering in some buildifier output that got converted into CMake or something... but no, this was manually written. I'll just rewrite so `root` is before `root_suffix`.